src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+
+        break;
+      case 'removeProperties':
+        for (const keyToRemove of action.keysToRemove) {
+          delete state[keyToRemove];
+        }
+
+        break;
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+
+        break;
+      default:
+        break;
```

Comment:
it would be great to `throw new Error` on default case with message about wrong action type